examples/structured_data/classification_with_tfdf.py

```diff
@@ -383,22 +387,22 @@ def call(self, inputs):
         # Cast the inputs int64 a tensor.
         inputs = tf.cast(inputs, tf.dtypes.int64)
         # Lookup positive frequencies for the input feature values.
-        positive_fequency = tf.cast(
+        positive_frequency = tf.cast(
             tf.gather_nd(self.positive_frequency_lookup, inputs),
             dtype=tf.dtypes.float32,
         )
         # Lookup negative frequencies for the input feature values.
-        negative_fequency = tf.cast(
+        negative_frequency = tf.cast(
             tf.gather_nd(self.negative_frequency_lookup, inputs),
             dtype=tf.dtypes.float32,
         )
         # Compute positive probability for the input feature values.
-        positive_probability = positive_fequency / (
-            positive_fequency + negative_fequency
+        positive_probability = positive_frequency / (
+            positive_frequency + negative_frequency
         )
         # Concatenate and return the looked-up statistics.
         return tf.concat(
-            [positive_fequency, negative_fequency, positive_probability], axis=1
+            [positive_frequency, negative_frequency, positive_probability], axis=1
```

Comment:
Can you address the comment starting with:

> This is not really an embedding. Instead, it will be the output of the linear model trained with categorical buckets. So it is fundamentally very similar to target mean.

in https://github.com/keras-team/keras-io/pull/782

Reply:
OK I added a ReLU non-linearity to the encoder, and called the it NN model, rather than linear model